frontends/election-manager/src/components/election_manager.tsx

```diff
@@ -159,10 +159,7 @@ export function ElectionManager(): JSX.Element {
             <DefinitionContestsScreen allowEditing={false} />
           </Route>
           <Route exact path={routerPaths.ballotsList}>
-            <BallotListScreen />
-          </Route>
-          <Route exact path={routerPaths.printedBallotsReport}>
-            <PrintedBallotsReportScreen />
```

Comment:
It isn't possible to print a printed ballots report as a system admin, so this route isn't needed